ontology/yaml/resources/ELECTRICAL/entity_types/ABSTRACT.yaml

```diff
@@ -51,6 +51,44 @@ IOBM:
   - bypass_phase2_neutral_voltage_sensor
   - bypass_phase3_neutral_voltage_sensor
   
+
+PWM:
+  description: "Very basic power and enrgy monitoring only."
+  is_abstract: true
+  uses:
+  - power_sensor
+  - energy_accumulator
+  - powerfactor_sensor 
+  - apparent_power_sensor
+  - battery_voltage_sensor  
```

Comment:
dont need this on here. move it elsewhere, otherwise this very basic power and energy monitoring device becomes a battery device, with limited reusability.